pkg/reconciler/namespace/namespace.go

```diff
@@ -128,8 +128,7 @@ func (c *Controller) updateSchedulableCondition(ctx context.Context, ns *corev1.
 		})
 	}
 	return c.patchStatus(ctx, ns, func(conditionsSetter conditions.Setter) {
-		conditions.MarkFalse(conditionsSetter, NamespaceScheduled, NamespaceReasonUnschedulable,
-			conditionsv1alpha1.ConditionSeverityNone, // NamespaceCondition doesn't support severity
+		conditions.MarkFalse(conditionsSetter, NamespaceScheduled, NamespaceReasonUnschedulable, conditionsv1alpha1.ConditionSeverityNone, // NamespaceCondition doesn't support severity
```

Comment:
Please keep the ConditionSeverityNone line and its comment on the same line for readability.